cli/marketplace/build.py

```diff
@@ -668,7 +668,7 @@ def sphinx_quickstart(
     click.echo("[Sphinx] Done quickstart")
 
 
-def build_temp_docs(temp_root, temp_docs):
+def build_temp_docs(temp_root, temp_docs, source_dir):
```

Comment:
just add a docstring as well for the `source_dir`:
`:param source_dir: Path to the source directory to build the marketplace from`